packages/components/src/call-out/index.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { Button, __experimentalText as Text } from '@wordpress/components';
+import { CallOut, CallOutIllustration } from './';
+
+const meta: Meta< typeof CallOut > = {
+	title: 'CallOut',
+	component: CallOut,
+	tags: [ 'autodocs' ],
+	parameters: {
+		actions: { argTypesRegex: '^on.*' },
+		// Prevent flickering + automatic menu closing in compact mode.
+		layout: 'fullscreen',
+	},
+	decorators: [
+		( Story ) => (
+			<div style={ { paddingInlineStart: '1rem', paddingBlockStart: '1rem' } }>
+				<Story />
+			</div>
+		),
+	],
+};
+
+export default meta;
+type Story = StoryObj< typeof CallOut >;
+
+export const Default: Story = {
+	args: {
+		title: 'Let our WordPress.com experts build your site!',
+		children: (
+			<Text as="p" variant="muted">
+				Hire our dedicated experts to build a handcrafted, personalized website. Share some details
+				about what you’re looking for, and we’ll make it happen.
+			</Text>
+		),
+		illustration: (
+			<CallOutIllustration
+				src="https://live.staticflickr.com/3277/2938134470_c807dc3e47_b.jpg"
+				alt="Sweet eyed kitty"
+			/>
```

Comment:
By using a special `CallOutIllustration` component we're able to set the necessary `object-fit` rules that allow the illustration to always be perfectly vertically aligned with the content on the left.

I'm not planning on implementing the "full bleed" version of the illustration, but having a special component should give us the power we need to implement this too when the time comes.

What do you think?

Reply:
Could we apply the additional styles by adding child-selectors assuming this would be passed as an `img` ?

e.g.

```scss
.call-out-illustration img {
  // ...
}
```